lib/manager/npm/extract/pnpm.ts

```diff
@@ -0,0 +1,104 @@
+import yaml from 'js-yaml';
+import { logger } from '../../../logger';
+import {
+  findLocalSiblingOrParent,
+  getSiblingFileName,
+  localPathExists,
+  readLocalFile,
+} from '../../../util/fs';
+import type { PackageFile } from '../../types';
+import { matchesAnyPattern } from './utils';
+
+export async function extractPnpmFilters(
+  fileName: string
+): Promise<string[] | null> {
+  try {
+    const contents = yaml.safeLoad(await readLocalFile(fileName, 'utf8'), {
+      json: true,
+    }) as any;
```

Comment:
Please create some types to `./types.ts` instead of using `any`. You only need to define the props you are using here.

Reply:
added in c5b159c37e3b7aed51bd6071bfe72a68977f139e